packages/client/src/App.sass

```diff
@@ -3,50 +3,96 @@
 @import "@fontsource/dm-mono"
 $navWidth: 260px
 
-// Set your brand colors
-$bodyBg: #162c4e
-$bodyColor: #bbc6ce
-$primaryPurple: #7464F0
-$backgroundPurple: #9B8FF9
-$darkPurple: #000E64
-
-// Update Bulma's global variables
-$grey-dark: $bodyColor
-$grey-darker: $bodyColor
-$grey-light: $bodyColor
-$link: white
-$primary: $primaryPurple
-$light: $backgroundPurple
+/**********************Initial variables - Colors *******************/
+$black: #000000
+$white: #FFFFFF
+$success-green: #18AA01
+$warning-orange: #F26300
+$error-red: #E30C0C
+
+//grey:
+$grey-primary: #939393
+$grey-dark: #6D6D6D
+$grey-secondary: #DDDDDD
+$grey-tertiary: #EEEEEE
+
+//purple
+$purple-primary: #7464F0
+$purple-dark: #000E64
+$purple-secondary: #9B8FF9
+$purple-tertiary: #DFDBFF
+$purple-background: #F0F0FE
+
+//green
+$green-primary: #DCFABF
+$green-dark: #A4C881
+
+//yellow
+$yellow-primary: #D6D71C
```

Comment:
This naming strategy corresponds with the design specs(https://www.figma.com/file/sKtcz6ytBmNNDNW7XU8ea5/Vessel%3A-User-Experience?node-id=153%3A61271) since we have multiple shades for one color, I thought it would be clearer make it a convention (dark ->primary->secondary->tertiary->background) primary means it's the color on the second column of the color row